eternalcombat-plugin/src/main/java/com/eternalcode/combat/config/implementation/PluginConfig.java

```diff
@@ -118,6 +118,11 @@ public static class Settings extends OkaeriConfig {
         })
         public Duration combatTimerDuration = Duration.ofSeconds(20);
 
+        @Comment({
+            "# Should plugin kill players on combat log?",
+        })
+        public boolean shouldPlayerBeKilledOnCombatLog = true;
```

Comment:
You mean "logout" sure? Also, I think we can shorten it, e.g. `killOnCombatLogout`.

Reply:
I agree, the current name is too long.